ci/use_conda_packages_from_prs.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+# Copyright (c) 2025, NVIDIA CORPORATION.
+
+# download CI artifacts
+LIBRMM_CHANNEL=$(rapids-get-pr-conda-artifact rmm 1858 cpp)
+RMM_CHANNEL=$(rapids-get-pr-conda-artifact rmm 1858 python)
+
+# make sure they can be found locally
+conda config --system --add channels "${LIBRMM_CHANNEL}"
+conda config --system --add channels "${RMM_CHANNEL}"
+RATTLER_CHANNELS=("-c file://${RMM_CHANNEL}" "-c file://${LIBRMM_CHANNEL}" "${RATTLER_CHANNELS[@]}")
```

Comment:
Seeing [this CI error]( https://github.com/rapidsai/cudf/actions/runs/13827639762/job/38685496112?pr=18260#step:9:516 ):

```
Error:   × invalid channel name: ' file:///tmp/tmp.d4hHb9ziCZ'
```

Looks like the flag and value need to be separate elements in the array

```suggestion
RATTLER_CHANNELS=("-c" "file://${RMM_CHANNEL}" "-c" "file://${LIBRMM_CHANNEL}" "${RATTLER_CHANNELS[@]}")
```